.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+
+<< Describe the changes >>
+
+Closes issue:
+
+## Reviewer checks
```

Comment:
Minor thing, but I'd recommend using the underline form of headings for these templates:

```md
heading
---
```

Because that ensures tools like hub won't treat the heading as a comment: https://github.com/github/hub/issues/1377